package.json

```diff
@@ -118,5 +120,6 @@
     },
     "imports": {
         "#test-helpers": "./tests/test-helpers.js"
-    }
+    },
+    "packageManager": "pnpm@9.15.4+sha512.b2dc20e2fc72b3e18848459b37359a32064663e5627a51e4c74b2c29dd8e8e0491483c3abb40789cfd578bf362fb6ba8261b05f0387d76792ed6e23ea3b1b6a0"
```

Comment:
can you please revert it - we just don't want to force contributors to use pnpm - actually npm was used in the github ci :)

Reply:
Ah yes sorry. That wasn't supposed to be commited